From 350682f014b9559c5d3f4308dc8646def5aefae5 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Thu, 19 Jul 2012 17:48:15 -0400
Subject: [PATCH] Remove an unused case in the notification action cb

The 'update-all-packages' action is no longer hooked up to
any notification anymore, and the code for dealing with it
was using pk_task_update_system_async, which has been removed
from the PackageKit libraries.
---
 plugins/updates/gsd-updates-manager.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/plugins/updates/gsd-updates-manager.c b/plugins/updates/gsd-updates-manager.c
index fd3bd00..84003b5 100644
--- a/plugins/updates/gsd-updates-manager.c
+++ b/plugins/updates/gsd-updates-manager.c
@@ -242,14 +242,6 @@ libnotify_action_cb (NotifyNotification *notification,
                 }
                 goto out;
         }
-        if (g_strcmp0 (action, "update-all-packages") == 0) {
-                pk_task_update_system_async (manager->priv->task,
-                                             manager->priv->cancellable,
-                                             NULL, NULL,
-                                             (GAsyncReadyCallback) update_packages_finished_cb,
-                                             manager);
-                goto out;
-        }
         if (g_strcmp0 (action, "clear-offline-updates") == 0) {
                 clear_offline_updates_message ();
                 goto out;
-- 
1.7.11.2

